src/github/pullRequestOverview.ts

```diff
@@ -189,17 +190,17 @@ export class PullRequestOverviewPanel {
 	 * @param timelineEvents The timeline events
 	 * @param commits All review commits
 	 */
-	private fixCommitAttribution(timelineEvents: TimelineEvent[], commits: Commit[]): void {
-		const commitEvents: CommitEvent[] = (<CommitEvent[]>timelineEvents.filter(event => event.event === EventType.Committed));
+	private fixCommitAttribution(timelineEvents: TimelineEvent[], commits: Github.PullRequestsGetCommitsResponseItem[]): void {
+		const commitEvents = (<CommitEvent[]>timelineEvents.filter(event => event.eventType === EventType.Committed));
 		for (let commitEvent of commitEvents) {
 			const matchingCommits = commits.filter(commit => commit.sha === commitEvent.sha);
 			if (matchingCommits.length === 1) {
 				const author = matchingCommits[0].author;
 				// There is not necessarily a GitHub account associated with the commit.
 				if (author !== null) {
-					commitEvent.author.avatar_url = author.avatar_url;
-					commitEvent.author.login = author.login;
-					commitEvent.author.html_url = author.html_url;
+					commitEvent.authorExtra.avatar_url = author.avatar_url;
```

Comment:
this should be left as author so that we update the existing object, authorExtra is undefined

Reply:
I'm now extending the author property properly now, not sure why I didn't do it in the first place. :D